include/blmc_drivers/devices/motor_board.hpp

```diff
@@ -19,6 +19,7 @@
 #include "blmc_drivers/devices/can_bus.hpp"
 #include "blmc_drivers/devices/device_interface.hpp"
 
+namespace rt = real_time_tools;
```

Comment:
I think it would be better to move this inside the `blmc_drivers` namespace below.

Reply:
This is still there